content/en/docs/collector/custom-collector.md

```diff
@@ -20,13 +20,12 @@ builder][ocb] (or `ocb` for short) to assist people in assembling their own
 distribution, making it easy to build a distribution that includes their custom
 components along with components that are publicly available.
```

Comment:
Let's shorten this paragraph a little.

You can use the [OpenTelemetry Collector
Builder][ocb] (or `ocb` for short) to assemble your own
distribution that includes custom
components, upstream components, and other publicly available components.